### ISSUE TITLE
squish tests fail on master, update the version of squish used on master?

### ISSUE THREAD

**Issue by TheOneRing** [owncloud/client#9240]
```
/drone/src/build-GUI-tests/bin/owncloud: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /drone/src/build-GUI-tests/bin/owncloud)
```

_Originally posted by @erikjv in https://github.com/owncloud/client/issues/9184#issuecomment-979003582_

**Comment by individual-it** [comment]
We can update squish, but I think that is more about having a that version of the library actually to start the client.
Do different versions of the client need different versions of glibc?

**Comment by TheOneRing** [comment]
Should not be the case.
@fmoc ^

**Comment by Talank** [comment]
Do we need to upgrade qt5 to version `5.15` to compile client? I can compile the code on 2.9 and 2.10 but not the master! getting error duting the `cmake`
```
Could not find a configuration file for package "Qt5" that is compatible
  with requested version "5.15".

  The following configuration files were considered but not accepted:

    /usr/lib/x86_64-linux-gnu/cmake/Qt5/Qt5Config.cmake, version: 5.12.8
    /lib/x86_64-linux-gnu/cmake/Qt5/Qt5Config.cmake, version: 5.12.8
```

**Comment by individual-it** [comment]
CI uses Qt 5.15
![grafik](https://user-images.githubusercontent.com/2425577/143430449-7766ce09-8fa6-4156-b914-e6235a034e5b.png)

**Comment by TheOneRing** [comment]
Yes master requires Qt 5.15

**Comment by individual-it** [comment]
Then this becomes a bigger task, we need to use different versions of squish in master and in the other repos.
The squish docker container needs to get tags, according to the used Qt version

**Comment by individual-it** [comment]
I guess we need to make similar changes like @fmoc done in https://github.com/owncloud-ci/client/pull/13/files

**Comment by individual-it** [comment]
WIP PR: https://github.com/owncloud-ci/squish/pull/26

**Comment by individual-it** [comment]
strange even with the chages from  https://github.com/owncloud-ci/client/pull/13/files I get Qt 5.14

**Comment by individual-it** [comment]
Qt 5.15 installed by using Ubuntu 21.10 and the client compiles fine but 21.10 comes with Python 3.9 and squish only supports 3.8
```
The Squish package has no support for Python 3.9 and cannot be configured for this interpreter.
The package contains support for the following Python version(s): 2.7.x & 3.8.x
Please contact support@froglogic.com to inquire about support for Python 3.9 if you want to use that version.

**Comment by individual-it** [comment]
also building python3.8 from source does not work
```
Python has been statically built and will not work for Squish
```

**Comment by individual-it** [comment]
reply from froglogic about python 3.9 support
```
We apologize for your inconvenience but there is no schedule for binary packages supporting Python 3.9 at the moment.
```

I will try with Ubuntu 21.04

**Comment by individual-it** [comment]
now trying to get Qt 5.15 on Ubuntu 20.04

**Comment by individual-it** [comment]
next problem I need a new version of glibc

**Comment by TheOneRing** [comment]
So for compiling python dynamically there should be a solution
did the first mix work besides python?

**Comment by TheOneRing** [comment]
How did you build python? I usually use https://github.com/pyenv/pyenv not sure how they link however.

**Comment by individual-it** [comment]
@TheOneRing its the same as building it yourself its statically build and squish does not like it

**Comment by individual-it** [comment]
>  did the first mix work besides python?

yes if we can get a dynamically linked python 3.8 on ubuntu 21.10 that would solve it all
there is a ppa, but that does not have that combination https://github.com/deadsnakes/

**Comment by TheOneRing** [comment]
https://github.com/python/cpython/blob/main/configure#L1708
Should be possible.

**Comment by individual-it** [comment]
@TheOneRing that seems to work, thank you I haven't seen that option. Now I need to bundle it all

**Comment by individual-it** [comment]
squish and the tests start, but:
- client takes a very long time to wait for initial sync
- some UI actions don't work

more work to do

**Comment by individual-it** [comment]
xfce4 does not start correctly

**Comment by individual-it** [comment]
maybe its a tigerVNC issue, I will try to rebuild with ubuntu 20.04

**Comment by individual-it** [comment]
it all works when I start the docker container with `--priviledged`, but we cannot do that in drone

**Comment by individual-it** [comment]
it can be fixed by using https://github.com/gdraheim/docker-systemctl-replacement/ let's see what problem will be next

**Comment by individual-it** [comment]
with Ubuntu 21.04 it works without the `--priviledged` switch

**Comment by individual-it** [comment]
@fmoc @TheOneRing It seem to work with Ubuntu 20.04 see https://github.com/owncloud-ci/client/pull/14 & https://github.com/owncloud-ci/squish/pull/26

so tests generally run see https://drone.owncloud.com/owncloud/client/9885/5/13
BUT: The client needs a long time to get going. After starting it take around 1min to get out of the "waiting" phase. It sends multiple quota requests in the background and then all comes green. For the tests that's a very long time.

**Comment by individual-it** [comment]
the test problem should be fixed by merging owncloud-ci/client#14 & owncloud-ci/squish#26

After that I expect the UI tests in master to start but they will run for ever and drone will timeout eventually because of https://github.com/owncloud/client/issues/9253

**Comment by individual-it** [comment]
would be fixed by #9302 that PR contains the fix for #9253 and also all fixes for the tests

**Comment by individual-it** [comment]
fixed by #9302
nightly also run stable now
